.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+image:
+  file: .devcontainer/Dockerfile
```

Comment:
This can also be `image: gitpod/workspace-dotnet`; they build that workspace by running the dotnet-install script for the latest SDK so it's always up to date.